src/main/java/core/basesyntax/service/ConvertReadDataService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Transaction;
+import java.util.List;
+
+public interface ConvertReadDataService {
+    List<Transaction> convertDataFromFile(List<String> dataFromFile);
```

Comment:
Think more about naming. The argument is **List<String>** Why do we have "file" in the method name and argument name? We do not know where did this List come from - file, mail, web...
```suggestion
    List<Transaction> getTransactions(List<String> data);
```